### ISSUE TITLE
MQTTConnect need userName

### ISSUE THREAD

**Issue by zhaoyouzhi** [Qatja/qatja-android#9]
![image](https://user-images.githubusercontent.com/17698787/39037729-ee796776-44b3-11e8-8254-426cd61537e5.png)

**Comment by zhaoyouzhi** [comment]
Pull Request

**Comment by agoransson** [comment]
I'm fixing this tonight. Will be release 0.1.21

**Comment by agoransson** [comment]
To use username and password simply set the username and password in methods before the connect:

```
// Default host is test.mosquitto.org (you should change this!)
client.setHost(host);

// Default mqtt port is 1883
client.setPort(1883);

// Set a unique id for this client-broker combination
client.setId(clientIdentifier);

// Set keep alive time
client.setKeepAlive(3000);

// Set username & password (optional)
client.setUsername("my-username");
client.setPassword("my-password");

// Open the connection to the MQTT server
client.connect();
```

**Comment by agoransson** [comment]
The fix is now published as 0.1.21.

If you need it quickly you can always temporarily add the local maven repo to your project.
```
maven { url 'https://dl.bintray.com/wetcat/Qatja' }
```
